docs/src/2.5.AddingInteractivity.mdx

```diff
@@ -33,20 +34,20 @@ Let's also enable manual camera movement, so we can see the duck better.
 
 ```js
 // use `defaultCameraState` to turn Worldview to uncontrolled component
-<Worldview defaultCameraState={{ ...DEFAULT_CAMERA_STATE, distance: 120 }}>
+<Worldview defaultCameraState={{ ...DEFAULT_CAMERA_STATE, distance: 160 }}>
 ```
 
 Next, let's add a `clickedObjectIds` state to keep track of the clicked spheres. We'll also use this `clickedObjectIds` state to generate cube obstacles.
 
 ```js
-const [clickedObjectIds, setClickedObjectIds] = useState(new Set());
+const [clickedObjectIds, setClickedObjectIds] = React.useState([]);
```

Comment:
I think it's common to `import React, { useState }` and just use `useState` rather than `React.useState`